Lib/glyphsLib/builder/glyph.py

```diff
@@ -35,9 +35,16 @@ def to_ufo_glyph(self, ufo_glyph, layer, glyph_data):
     if last_change is not None:
         ufo_glyph.lib[GLYPHLIB_PREFIX + 'lastChange'] = to_ufo_time(last_change)
     color_index = glyph_data.color
-    if color_index is not None and color_index >= 0:
+    if color_index is not None:
         ufo_glyph.lib[GLYPHLIB_PREFIX + 'ColorIndex'] = color_index
-        ufo_glyph.lib[PUBLIC_PREFIX + 'markColor'] = GLYPHS_COLORS[color_index]
+        color_tuple = None
+        if isinstance(color_index, list):
+            color_tuple = ','.join(['{0:.4f}'.format(i/255.0)
```

Comment:
Note that we are using python 3 division operator (or if we don’t we should, by importing it `from __future__`.
You need to use two slashes for integer division; one slash is always float division.

Reply:
The join method of str accept an iterable, so you can pass it a generator expression directly without needing to create a temporary list (with a list comprehension).
“”.join(x for x in something)